a1/README.md

```diff
@@ -3268,6 +3269,95 @@ Use [Gulp](http://gulpjs.com) or [Grunt](http://gruntjs.com) for creating automa
 
 **[Back to top](#table-of-contents)**
 
+## Low Level Functions
+
+Angular 1.x ships with some [helper functions](https://docs.angularjs.org/api/ng/function) that are found within JavaScript natively. In such cases we should favor usage of native JavaScript functions instead of Angular's helper functions. In other cases where functionality is not found in JavaScript natively, we should turn to utility libraries such as [lodash](https://lodash.com/docs) to fill-in the gaps.
```

Comment:
That's a very good point. I wonder how to balance that with the fact that if one ever wants to upgrade to Angular 2, he or she will have to get rid of these method calls anyway.

Just mentioning this since eventual upgrade to Angular 2 is one of the _why's_.